src/weekly/query.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { fetchConfig } from './../lib/config'
+import type {Config} from './../lib/config'
+import {Zuora} from './../lib/Zuora'
+import type {Query} from './../lib/Zuora'
+import moment from 'moment'
+
+type input = {
+  deliveryDate: ?string,
+  deliveryDateDaysFromNow: ?number
+}
+async function queryZuora (deliveryDate, config: Config) {
+  const formattedDate = deliveryDate.format('YYYY-MM-DD')
+ // const deliveryDay = deliveryDate.format('dddd')
+  const zuora = new Zuora(config)
+  const subsQuery: Query =
+    {
+      'name': 'WeeklySubscriptions',
+      'query': `
+        SELECT
+            RateplanCharge.quantity,
+            Subscription.Name,
+              SoldToContact.Address1,
+              SoldToContact.Address2,
+              SoldToContact.City, 
+              SoldToContact.Country, 
+              SoldToContact.Title__c,
+              SoldToContact.FirstName, 
+              SoldToContact.LastName, 
+              SoldToContact.PostalCode, 
+              SoldToContact.State
+          FROM
+            rateplancharge
+          WHERE
+           (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
+           ProductRatePlanCharge.ProductType__c is not null AND
+           Product.Name like '%Weekly%' AND
+           RatePlanCharge.EffectiveStartDate <= '${formattedDate}' AND
+           RatePlanCharge.EffectiveEndDate >= '${formattedDate}' AND
```

Comment:
This might have to change later. Some one-off GW subs have an effective end date in the past and still need to be fulfilled.